lib/routes/drivers/routes.js

```diff
@@ -0,0 +1,141 @@
+import { Sequelize } from 'sequelize';
+import Joi from '@hapi/joi';
+import {
+    addDriver,
+    updateDriver,
+    addDriverLocation,
+    acceptBooking,
+    updateBookingAttempts
+} from 'daos/driverDao';
+import {
+    emailSchema,
+    idAllowedSchema,
+    stringSchema
+} from 'utils/validationUtils';
+
+module.exports = [
+    {
+        method: 'POST',
+        path: '/add',
```

Comment:
nit: I think the route should just be [POST] `/drivers`. Their is no spec for REST but almose understood everywhere that `/drivers` POST method creates a driver.

Reply:
Done.